docs-compose/overwatch_docs/content/DataEngineer/Pipeline_Management.md

```diff
@@ -77,22 +77,6 @@ so be sure to read the rest of this page before altering this table.
 The Pipeline Report is very useful for identifying issues in the pipeline. Each module, each run is detailed here. 
 The structure of the table is outlined [below](#pipeline_report-structure).
 
-A *pipReport* view has been created atop the pipeline_report table to simplify reviewing the historical runs. The 
-query most commonly used is below. If you are asked for the pipReport, please provide the output from the following
-
-**SCALA**
-```scala
-table("overwatch_etl.pipReport")
-  .filter('organization_id === "<workspace_id>") // if you want to see the pipReport for only a specific workspace
-  .orderBy('Pipeline_SnapTS.desc)
-```
-**SQL**
-```sql
-select * from overwatch_etl.pipReport
-    where organization_id = '<workspace_id>' -- if you want to see the pipReport for only a specific workspace
-    order by Pipeline_SnapTS desc
-```
-
 ### Pipeline_Report Structure
```

Comment:
I know you didn't make changes to the pipeline_report docs, but they are wrong, can you tackle them as part of this PR as well? For example, it doesn't mention workspace_name as a column and a key, or writeOpsMetrics, and it still has dataFrequency and recordsAppended, both which are deprecated columns.

Reply:
done